tests/components/atag/test_atag_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Tests for the Atag config flow."""
+from unittest.mock import PropertyMock, patch
+
+from asynctest import CoroutineMock
+
+from homeassistant import data_entry_flow
+from homeassistant.components.atag import config_flow
+from homeassistant.const import CONF_DEVICE, CONF_HOST, CONF_PORT
+
+from tests.common import MockConfigEntry
+
+FIXTURE_USER_INPUT = {
+    CONF_HOST: "127.0.0.1",
+    CONF_PORT: 10000,
+}
+FIXTURE_COMPLETE_ENTRY = FIXTURE_USER_INPUT.copy()
+FIXTURE_COMPLETE_ENTRY[CONF_DEVICE] = "device_identifier"
+
+
+async def test_show_form(hass):
+    """Test that the form is served with no input."""
+    flow = config_flow.AtagConfigFlow()
+    flow.hass = hass
+
+    result = await flow.async_step_user(user_input=None)
```

Comment:
Please use the core interfaces to start the tests.
```suggestion
    result = await hass.config_entries.flow.async_init(
        DOMAIN, context={"source": config_entries.SOURCE_USER}
    )
```